Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorUtils.cpp

```diff
@@ -42,22 +42,19 @@ GRefPtr<GBytes> backendCommands()
 #if PLATFORM(WPE)
     static std::once_flag flag;
     std::call_once(flag, [] {
-        const char* libDir = PKGLIBDIR;
+        const char* dataDir = PKGDATADIR;
+        GUniqueOutPtr<GError> error;
 
-        // Probably no need for a specific env var here. Assume the inspector resources.so file is
-        // in the same directory as the injected bundle lib, for developer builds.
-        const char* path = g_getenv("WEBKIT_INJECTED_BUNDLE_PATH");
+        const char* path = g_getenv("WEBKIT_INSPECTOR_RESOURCES_PATH");
```

Comment:
afaics none of our tooling is setting this env var. Can you follow-up about this?

Reply:
No I don't think the env var is set anywhere, the idea was just to let people pass in the variable if the developer mode is enabled, should we add it to scripts for WPE API?

Reply:
Yes, in a similar manner as `WEBKIT_INJECTED_BUNDLE_PATH` (grep in Tools :))

Reply:
Just checking in with you guys after this merge: https://github.com/WebKit/WebKit/pull/35403
There's a bunch few other places where `WEBKIT_INJECTED_BUNDLE_PATH` is set, like in the minibrowser and the tests, should I also add `WEBKIT_INSPECTOR_RESOURCES_PATH` in a similar way?
(also @carlosgcampos)

Reply:
@siwei-li Yes, please go ahead and add the environment variable in the places where it's missing.